### PR TITLE
add avatar and badge components.

### PR DESCRIPTION
**Pull request by AnkurChunekar**

### CODE REVIEW

**Review by kart-c — APPROVED** [review]
looks good. Good work on the comments Ankur

**Review by Abhijeet43 — APPROVED** [review]
Hey Ankur, the code looks fine to me.

**Review by Husain01 — APPROVED** [review]
All the components are perfectly done. Good going.

**Review by RakshaPawar108 — COMMENTED** [review]
The components look great and beautiful. Code is also pretty neat and commented. Great work!

**Review by RakshaPawar108 — APPROVED** [review]

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+/* || Standard Avatar Component ends */
+
+/* || Different Sizes for Avatar starts */
+
+.avatar.avatar-xs {
+  height: 3.2rem;
+  width: 3.2rem;
+}
+.avatar.avatar-sm {
```
Comment (Meghana-sk) [inline components/avatar/avatar.css:42]: Ankur, good work. Here you can add one empty line  to maintain line gaps after each styles for classes.

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  height: 4.8rem;
+  width: 4.8rem;
+}
+.avatar.avatar-md {
```
Comment (Meghana-sk) [inline components/avatar/avatar.css:46]: line gap could be given

File: components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  height: 6.4rem;
+  width: 6.4rem;
+}
+.avatar.avatar-lg {
```
Comment (Meghana-sk) [inline components/avatar/avatar.css:50]: line gap could be given

**Review by Meghana-sk — COMMENTED** [review]
Good work

**Pull request merged**